packages/snowflake/src/lib/Snowflake.ts

```diff
@@ -129,7 +129,8 @@ export class Snowflake {
 	public static compare(a: string | bigint, b: string | bigint): -1 | 0 | 1 {
 		if (typeof a === 'bigint' || typeof b === 'bigint') {
 			if (typeof a === 'string') a = BigInt(a);
-			else if (typeof b === 'string') b = BigInt(b);
+			if (typeof b === 'string') b = BigInt(b);
```

Comment:
This is unnecessary, because the check at 130 ensures that either `a` or `b` is `bigint`, so only one may be `string`. You're free to try to prove this with unit tests, although I think those exist already.

Reply:
<img width="1004" alt="image" src="https://user-images.githubusercontent.com/4019718/230916079-db039002-9c75-4ee5-8af1-39c87c27addf.png">